app/scripts/controllers/SysinfoController.coffee

```diff
@@ -103,5 +105,16 @@ angular.module('neo4jApp.controllers')
       $scope.autoRefresh = !$scope.autoRefresh
       refreshLater()
 
+    $scope.isCurrentConnection = (addresses) ->
+      if Settings.useBolt
+        boltHost = Settings.boltHost || $rootScope.boltHost
```

Comment:
It would be better to reuse https://github.com/neo4j/neo4j-browser/blob/2.0/app/scripts/services/UtilityBolt.coffee#L246 to get the Bolt host, I think.